test/functionalTest/harnessFunctions.sh

```diff
@@ -90,49 +90,45 @@ function dMsg()
 
 # ------------------------------------------------------------------------------
 #
-# dbInit - 
+# brokerStart CB - 
 #
 function dbInit()
 {
-  role=$1
-  
-  host="${CB_DATABASE_HOST}"
-  if [ "$host" == "" ]
-  then
-    host="localhost"
-  fi
-  
-  port="${CB_DATABASE_PORT}"
-  if [ "$port" == "" ]
-  then
-    port="27017"
-  fi
-  
-  db=$host:$port/$1
+  role=$1  
+  db=$1
```

Comment:
Probably when I started to develop the test was with an old version.
I have proved with the file that is on the  develop branch, and it works, so I  have restored the same file.

Fixed in 0a207f6